regenerate-rust-annotations.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+set -e
+set -x
+
+cargo build -p capnpc
+capnp compile -otarget/debug/capnpc-rust-bootstrap:capnp/src capnpc/rust.capnp --src-prefix capnpc/
```

Comment:
What's the reasoning behind adding a new script here, rather than using the existing `regenerate-capnp-schema-code.sh`?

Reply:
Just assumed that would be how you wanted it based on the filename and the existing pattern

Reply:
This is irrelevant because I no longer need the annotation type